src/Agent.Plugins/PipelineArtifact/PipelineArtifactPlugin.cs

```diff
@@ -35,8 +35,8 @@ public async Task RunAsync(AgentTaskPluginExecutionContext context, Cancellation
 
         // Process the command with preprocessed arguments.
         protected abstract Task ProcessCommandInternalAsync(
-            AgentTaskPluginExecutionContext context, 
```

Comment:
@jtpetty , we should run one of these whitespace cleanup passes over the whole repo in one pass. 
Thanks for cleaning up things like this, but it does clutter up reviews.

Reply:
yeah, sorry about that. I have VSCode set to clean up whitespace, but it does the whole file and not on just the lines I edited.